todo_service/migration/001_create_todo_table.up.sql

```diff
@@ -0,0 +1 @@
+-- Create TODO TABLE
```

Comment:
lets call this directory `migrations` instead of `migration`

Reply:
then i would have to use plural on the all folder names like apis instead of api right?

Reply:
Not necessarily, but yes that works as well. In general we would want to make package name clear and meaningful. So anything that represents it is a `collection` of something, should be denoted by a plural noun e.g migrations, models, services, apis.
Good that you identified other such packages as well.

For particularly API, it is not necessary because technically it is not a package, it's a directory that contains two packages. But since it contains a collection, we can call it apis, that's fine.

Reply:
Thanks for the appreciation and the explanation.